rules-topics/21_status-effects.md

```diff
@@ -17,6 +17,8 @@ Dispel Magic and Nullify Spell work on all effects with a magical source unless
 ---
 If a term is denoted as visible, it is something you are permitted to respond with if a player asks “What do I see?”
 
+If a player is in an unconscious state (Bleeding Out, Sleep, Dead, etc.) and they are asked “What do I see?” the proper response is “An unresponsive body”.
```

Comment:
The proper response is whatever the character's physical description is. "Unresponsive" isn't a visible effect or state.

"What do I see?"
"Elf as I appear."

"What do I see?"
"Bark-covered humanoid recognizable as Verse."

Reply:
This was chosen specifically because it grouped dead, sleep, bleeding out together in a that is not already a description.
If you are dead you don't want to say unconscious because that is not entirely true and you can't say dead with a "What do I see call".  We can change this to something else but I am not sure a better phrase.